bedrock/careers/templates/careers/base.html

```diff
@@ -4,74 +4,50 @@
-
-    {# js #}
-    {{ js_bundle('careers-dnt-helper') }}
-    <script src="{{ static('js/careers/protocol-base.min.js') }}"></script>
-    <script src="{{ static('js/careers/protocol-utils.min.js') }}"></script>
-    <script src="{{ static('js/careers/protocol-supports.min.js') }}"></script>
-
-    {# icons #}
-    <link rel="shortcut icon" type="image/png" href="{{ static('img/careers/favicon.ico') }}"/>
-
-    {# feeds #}
-    <link rel="alternate" type="application/rss+xml"
-          href="{{ url('careers.feed') }}" title="Current Mozilla job openings" />
-
-    {% include 'includes/google-analytics.html' %}
-
-  </head>
-  <body id="{% block body_id %}{% endblock %}" class="{% block body_class %}{% endblock %}"
-    {%- block body_data %}
-    {%- endblock %}>
-
-    {% include "careers/includes/header.html" %}
-
-    <div id="wrapper" class="mzp-l-content mzp-t-mozilla">
-      <main role="main">
-
-        {% block content %}{% endblock %}
-
-      </main>
-    </div><!-- /#wrapper -->
-
-    {% block site_footer %}
-      {% include "careers/includes/footer.html" %}
-    {% endblock %}
-
-    {# js #}
-    {% block site_js %}
-      <script src="{{ static('js/careers/protocol-navigation.min.js') }}"></script>
-      <script src="{{ static('js/careers/protocol-menu.min.js') }}"></script>
-      <script src="{{ static('js/careers/base.js') }}"></script>
-    {% endblock %}
-
-  </body>
-</html>
+{% extends "base-protocol-mozilla.html" %}
+
+{% block page_title %}Mozilla Careers — Feel good about your work again{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block page_image %}{{ static('img/careers/moz-m.png') }}{% endblock %}
+
+{% block site_css %}
+  {{ css_bundle('careers') }}
+{% endblock %}
+
+{% block site_js %}
+  <!--[if !IE]><!-->
+    {{ js_bundle('lib') }}
+    <script src="{{ static('js/careers/libs/protocol-base.min.js') }}"></script>
+    <script src="{{ static('js/careers/libs/protocol-utils.min.js') }}"></script>
+    <script src="{{ static('js/careers/libs/protocol-supports.min.js') }}"></script>
+    {{ js_bundle('data') }}
+  <!--<![endif]-->
+{% endblock %}
+
+{% block extrahead %}
+  <link rel="alternate" type="application/rss+xml" href="{{ url('careers.feed') }}" title="Current Mozilla job openings" />
+{% endblock %}
+
+{% block site_header %}
+  {% include "careers/includes/header.html" %}
+{% endblock %}
+
+{% block content %}
+<div id="wrapper" class="mzp-l-content mzp-t-mozilla">
+  <main role="main">
+    {% block careers_content %}{% endblock %}
+  </main>
+</div>
+{% endblock %}
+
+{% block site_footer %}
+  {% include "careers/includes/footer.html" %}
+{% endblock %}
+
+{% block js %}
+  <script src="{{ static('js/careers/libs/protocol-navigation.min.js') }}"></script>
+  <script src="{{ static('js/careers/libs/protocol-menu.min.js') }}"></script>
+  {{ js_bundle('careers-base') }}
+{% endblock %}
+>>>>>>> 4b0f4a7d2 (Careers templates should extend base-protocol.html (Fixes #11027))
```

Comment:
@alexgibson Spotted this while exoploring some PRs. Simple deletion enough to remedy it?
```suggestion
```

Reply:
Bah, thanks